tasks/install_common.yml

```diff
@@ -1,13 +1,15 @@
 ---
 - name: Create the /tmp/efs-utils directory
   file:
+    mode: 0755
     path: /tmp/efs-utils
     state: directory
 
 - name: Download a tarball of the aws/efs-utils repository
   unarchive:
     src: https://github.com/aws/efs-utils/tarball/master
     dest: /tmp/efs-utils
+    mode: 0755
```

Comment:
Should this one be `0644`? I know the directory needs the execute bit, but do the tarball contents?

Reply:
@jsf9k confirmed that unless this mode matches the mode in the previous step, this task is not idempotent.  I added a comment about this in https://github.com/cisagov/ansible-role-amazon-efs-utils/pull/7/commits/57cdb3d4ede3def062bb879a607d332f0e8dfcbc.

Reply:
Ok, wasn't sure what the resolution on that was. All I saw was the commit to change the previous step to `0755`, but nothing about the interaction between eb174f0 and da6dd24 outside of the conversation in his comment about modes in his approval.